ir/ir.rs

```diff
@@ -513,6 +513,15 @@ pub enum ItemKind {
 
     /// Miscellaneous item. Perhaps metadata. Perhaps something else.
     Misc(Misc),
+
+    /// Program scope. Modules, namespaces, imported declarations.
+    Scope(Scope),
+
+    /// Subroutine item.
+    Subroutine(Subroutine),
```

Comment:
How is this intended to be different from `Code`? I would have expected parsing a `DW_TAG_subroutine` to create a `Code` IR item kind.